packages/amplify-cli/src/extensions/amplify-helpers/update-backend-config.js

```diff
@@ -17,21 +18,11 @@ function updateBackendConfigAfterResourceAdd(category, resourceName, options) {
   }
 }
 
-function updateBackendConfigDependsOn(category, resourceName, attribute, value) {
+function updateBackendConfigAfterResourceUpdate(category, resourceName, attribute, value) {
```

Comment:
Did you check on all the plugins referencing "updateBackendConfigDependsOn"?

Reply:
I searched for all referenced in the codebase and the ones in this PR are the only ones that came up